hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/hdds/scm/pipeline/TestRatisPipelineCreateAndDestroy.java

```diff
@@ -66,15 +67,43 @@ public void cleanup() {
   @Test(timeout = 180000)
   public void testAutomaticPipelineCreationOnPipelineDestroy()
       throws Exception {
+    int numOfDatanodes = 6;
+    init(numOfDatanodes);
+    // make sure two pipelines are created
+    waitForPipelines(2);
+    Assert.assertEquals(numOfDatanodes, pipelineManager.getPipelines(
+        HddsProtos.ReplicationType.RATIS,
+        HddsProtos.ReplicationFactor.ONE).size());
+
+    List<Pipeline> pipelines = pipelineManager
+        .getPipelines(HddsProtos.ReplicationType.RATIS,
+            HddsProtos.ReplicationFactor.THREE, Pipeline.PipelineState.OPEN);
+    for (Pipeline pipeline : pipelines) {
+      pipelineManager.finalizeAndDestroyPipeline(pipeline, false);
+    }
+    // make sure two pipelines are created
+    waitForPipelines(2);
+  }
+
+  @Test(timeout = 180000)
+  public void testAutomaticPipelineCreationDisablingFactorONE()
+      throws Exception {
+    conf.setBoolean(OZONE_SCM_PIPELINE_AUTO_CREATE_FACTOR_ONE, false);
     init(6);
     // make sure two pipelines are created
     waitForPipelines(2);
+    // No Factor ONE pipeline is auto created.
+    Assert.assertEquals(0, pipelineManager.getPipelines(
+        HddsProtos.ReplicationType.RATIS,
+        HddsProtos.ReplicationFactor.ONE).size());
+
     List<Pipeline> pipelines = pipelineManager
         .getPipelines(HddsProtos.ReplicationType.RATIS,
             HddsProtos.ReplicationFactor.THREE, Pipeline.PipelineState.OPEN);
     for (Pipeline pipeline : pipelines) {
       pipelineManager.finalizeAndDestroyPipeline(pipeline, false);
     }
+
     // make sure two pipelines are created
     waitForPipelines(2);
```

Comment:
I think this block (pipeline destroy) can be omitted, as the same is tested in `testAutomaticPipelineCreationOnPipelineDestroy`.

Reply:
I think the original intention was to test if pipeline could be auto created when previous pipelines were closed. Feel like it's still valid to keep future changes in pipeline creation consistence in terms of behaviors.